features/messages/impl/src/main/kotlin/io/element/android/features/messages/impl/forward/ForwardMessagesNode.kt

```diff
@@ -48,22 +67,44 @@ class ForwardMessagesNode @AssistedInject constructor(
     private val presenter = presenterFactory.create(inputs.eventId.value)
     private val callbacks = plugins.filterIsInstance<Callback>()
 
+    override fun resolve(navTarget: NavTarget, buildContext: BuildContext): Node {
+        val callback = object : RoomSelectEntryPoint.Callback {
+            override fun onRoomSelected(roomIds: List<RoomId>) {
+                presenter.onRoomSelected(roomIds)
```

Comment:
I do not think we have such pattern in the codebase yet, is it OK for you @ganfra ?

Reply:
I think in other scenarios we expose the View/Presenter/State/Events instead of a Node, but this is also ok I guess. But we should probably choose one pattern and sticks to it

Reply:
OK, I let it like this for now, we may want to revisit later.